src/Serilog.Sinks.Console/Sinks/SystemConsole/Rendering/ThemedMessageTemplateRenderer.cs

```diff
@@ -102,11 +102,12 @@ int RenderPropertyToken(PropertyToken pt, IReadOnlyDictionary<string, LogEventPr
 
         int RenderAlignedPropertyTokenUnbuffered(PropertyToken pt, TextWriter output, LogEventPropertyValue propertyValue)
         {
+            if(pt.Alignment == null) throw new ArgumentException("Invalid PropertyToken State. The PropertyToken should have a Alignment.", nameof(pt));
```

Comment:
Could simplify the message here to `"The PropertyToken should have a non-null Alignment."`?